bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/operation/WaterTransmissionVisitor.java

```diff
@@ -14,112 +15,114 @@
-                case 3:
-                    if (airMass <= 1.25) {
-                        name = ITCConstants.TRANSMISSION_LIB + "/" +
-                                file_name + "80_" + "10" +
-                                ITCConstants.DATA_SUFFIX;
-                    } else if (airMass > 1.25 && airMass <= 1.75) {
-                        name = ITCConstants.TRANSMISSION_LIB + "/" +
-                                file_name + "80_" + "15" +
-                                ITCConstants.DATA_SUFFIX;
-                    } else {
-                        name = ITCConstants.TRANSMISSION_LIB + "/" +
-                                file_name + "80_" + "20" +
-                                ITCConstants.DATA_SUFFIX;
-                    }
-                    break;
-                case 4:
-                    if (airMass <= 1.25) {
-                        name = ITCConstants.TRANSMISSION_LIB + "/" +
-                                file_name + "100_" + "10" +
-                                ITCConstants.DATA_SUFFIX;
-                    } else if (airMass > 1.25 && airMass <= 1.75) {
-                        name = ITCConstants.TRANSMISSION_LIB + "/" +
-                                file_name + "100_" + "15" +
-                                ITCConstants.DATA_SUFFIX;
-                    } else {
-                        name = ITCConstants.TRANSMISSION_LIB + "/" +
-                                file_name + "100_" + "20" +
-                                ITCConstants.DATA_SUFFIX;
-                    }
-                    break;
-                default:
-                    throw new IllegalArgumentException("unknown WV value");
-            }
-
-        } else {
-
-            final String _airmassCategory;
-            final String _transmissionCategory;
-
-            if (airMass <= 1.25)
-                _airmassCategory = "10";
-            else if (airMass > 1.25 && airMass <= 1.75)
-                _airmassCategory = "15";
-            else
-                _airmassCategory = "20";
-
-            switch (skyTransparencyWater) {
-                case 1:
-                    _transmissionCategory = "20_";
-                    break;
-                case 2:
-                    _transmissionCategory = "50_";
-                    break;
-                case 3:
-                    _transmissionCategory = "80_";
-                    break;
-                case 4:
-                    _transmissionCategory = "100_";
-                    break;
-                default:
-                    throw new IllegalArgumentException("unknown WV value");
-            }
-
-            name = "/HI-Res/" + abbrForSite(site) + wavelenRange + ITCConstants.TRANSMISSION_LIB + "/"
-                    + file_name + _transmissionCategory + _airmassCategory + ITCConstants.DATA_SUFFIX;
+        switch (instrument.getBands()) {
+            case VISIBLE:
+                switch (skyTransparencyWater) {
+                    case 1:
```

Comment:
This is a left-over I haven't cleaned up (yet). These values correspond to the WV bands. Same issue with CC and SB and IQ. Now that we have introduced those values into ITC we can replace these magic numbers at some point. It's on the still pretty long list for future refactorings..